eng/scripts/Language-Settings.ps1

```diff
@@ -610,4 +610,19 @@ function Get-java-DocsMsMetadataForPackage($PackageInfo) {
     PreviewReadMeLocation = 'docs-ref-services/preview'
     Suffix = ''
   }
+}
+
+function Validate-java-DocMsPackages { 
+  Param(
+    [Parameter(Mandatory = $true)]
+    [PSCustomObject]$PackageInfo,
+    [Parameter(Mandatory = $false)]
+    [string]$PackageSourceOverride,
+    [Parameter(Mandatory = $false)]
+    [string]$DocValidationImageId
+  ) 
+  $packageGroupId = $PackageInfo.Name
+  $packageVersion= $PackageInfo.Version
+  $p
```

Comment:
This looks like it is leftover.

Reply:
Or you intended to defined packageName.

Reply:
Why actually create local variables as opposed to just passing them in directly to the command below?